gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_0_20/schema-dashboards.yml

```diff
@@ -18,6 +18,7 @@ databaseChangeLog:
         - addNotNullConstraint:
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.